docs/docs/reference/configurations.md

```diff
@@ -97,6 +97,48 @@ Toggle CORS check function. Set `true` to check each request origin. Set `false`
 - Type: `Optional`
 - Default value: `true`
 
+### `SERVER_CORS_ENABLED`
+
+The server decides whether to enable the cors middleware and thus allow CORS requests. Enabling it also disables the same origin check in websocket Handler.
+
+- Type `Optional`
+- Default value: `true`
+
+### `SERVER_CORS_ALLOWED_ORIGIN`
+
+The server decides which origin to allow. The configuration is expected to space separated. Multiple values are supported. The value requies `SERVER_CORS_ENABLED` to be true to take effect. If you want to allow all host headers. You can pass `*` as the value.
+
+- Type `Optional`
+- Default Value `*`
+
+### `SERVER_CORS_ALLOWED_METHODS`
+
+The http methods allowed when it's a cross origin request. The http methods are expected to be space separated. 
+
+- Type `Optional`
+- Default Value `GET HEAD POST OPTIONS`
+
+### `SERVER_CORS_ALLOWED_HEADERS`
+
+The http request headers which are allowed when request is cross origin. The input expects to add any additional headers which is going to be sent by the client ex: `Authorization`. Headers which are essential for the functioning of Raccoon like Content-Type, Connection-Id & Group headers.
```

Comment:
can we mention that, the essential headers like Content-Type, Connection-Id & Group headers are already being added by default